app/views/recipes/_form.html.erb

```diff
@@ -11,71 +10,67 @@
+      <%= form.label :prep_time , class: "absolute text-sm text-teal-900 duration-300 transform -translate-y-4 scale-75 top-2 z-10 origin-[0] bg-gray-50 px-2 peer-focus:bg-gray-50 peer-focus:pb-0 peer-focus:px-2 peer-focus:text-teal-600 peer-placeholder-shown:scale-100 peer-placeholder-shown:-translate-y-1/2 peer-placeholder-shown:top-6 peer-focus:top-2 peer-focus:scale-75 peer-focus:-translate-y-4 rtl:peer-focus:translate-x-1/4 rtl:peer-focus:left-auto start-1" %>
     </div>
 
-    <div>
-      <%= form.label :cook_time %>
-      <%= form.text_field :cook_time, class: "block shadow rounded-md border border-gray-200 outline-none px-3 py-2 mt-2 w-full" %>
+    <div class="my-5 lg:my-2 basis-full lg:basis-auto relative">
+      <%= form.text_field :cook_time, class: "block rounded-lg px-2.5 pb-2.5 pt-5 w-full sm:w-4/5 text-sm text-gray-900 bg-gray-50  border border-teal-900 appearance-none focus:outline-none focus:ring-0 focus:border-teal-600 peer", placeholder: " " %>
+      <%= form.label :cook_time, class: "absolute text-sm text-teal-900 duration-300 transform -translate-y-4 scale-75 top-2 z-10 origin-[0] bg-gray-50 px-2 peer-focus:bg-gray-50 peer-focus:pb-0 peer-focus:px-2 peer-focus:text-teal-600 peer-placeholder-shown:scale-100 peer-placeholder-shown:-translate-y-1/2 peer-placeholder-shown:top-6 peer-focus:top-2 peer-focus:scale-75 peer-focus:-translate-y-4 rtl:peer-focus:translate-x-1/4 rtl:peer-focus:left-auto start-1" %>
     </div>
   </div>
 
-  <div class="flex flex-row flex-nowrap justify-between">
-    <%= form.label :rating, class:"mt-2" %>
-    <div class="rating mx-auto px-2">
-      <%= form.collection_radio_buttons(:rating, [[5],[4],[3],[2],[1]], :first, :last) do |star| %>
-        <%= star.radio_button %>
-        <%= star.label(class: "text-white") %>
-      <% end %>
-    </div>
+  <div class="my-5 lg:my-2 relative">
+    <%= form.collection_select :category_names, Category.all, :name, :name, {}, { data: { controller: "tom-select" }, multiple: true, placeholder:"Add Categories"} %>
   </div>
 
-  <div class="my-5 flex flex-row justify-between w-1/6">
-    <%= form.label "#{:is_favorite}?" %>
-    <%= form.check_box :is_favorite, rows: 4, class: "block shadow rounded-md border border-gray-200 outline-none px-2 py-2 mt-1" %>
-  </div>
-  
-  <div class="my-5">
-    <%= form.label :category_names, "Categories" %>
-    <%= form.collection_select :category_names, Category.all, :name, :name, {}, { data: { controller: "tom-select" }, multiple: true } %>
+  <div class="my-10 relative">
+    <%= form.text_area :description, class: "block rounded-lg px-2.5 pb-2.5 pt-5 w-full text-sm text-gray-900 bg-gray-50  border border-teal-900 appearance-none focus:outline-none focus:ring-0 focus:border-teal-600 peer", placeholder: " "  %>
```

Comment:
Assuming the controller was renamed and had a `resize` method, this is how it would be attached to this element with the initial size.
```suggestion
    <%= form.text_area :description, data: { controller: "textarea", action: "textarea#resize" }, rows: 2, class: "block rounded-lg px-2.5 pb-2.5 pt-5 w-full text-sm text-gray-900 bg-gray-50  border border-teal-900 appearance-none focus:outline-none focus:ring-0 focus:border-teal-600 peer", placeholder: " "  %>
```